skyportal/handlers/api/gcn.py

```diff
@@ -209,6 +221,24 @@ def delete(self, dateobs):
         return self.success()
 
 
+def add_contour(localization_id, request_handler):
+    try:
+        localization = (
+            Localization.query_records_accessible_by(request_handler.current_user)
+            .filter(
+                Localization.id == localization_id,
+            )
+            .first()
+        )
+        localization = get_contour(localization)
+        DBSession().add(localization)
+        DBSession().commit()
+    except Exception as e:
+        return log(f"Unable to generate contour: {e}")
```

Comment:
I would also print the localization ID here so that we can inspect it.

Reply:
Yes great idea. added